tests/integration/s3/test_s3.py

```diff
@@ -2704,13 +2704,12 @@ def test_put_object(self, s3_client, s3_bucket, snapshot):
         snapshot.match("get_object", response)
 
     @pytest.mark.aws_validated
-    @pytest.mark.xfail(
-        reason="failing sporadically with new HTTP gateway (only in CI)",
-    )
```

Comment:
Like we discussed, those presigned POST tests are very flaky, they could pass with arm64 then fail on amd64 and finally pass on integration_pro. Or fail on two of them but not the same tests failing. It seems sometimes there are not body in the request, and the signature/ETag returned is different, but no exception is raised before.